src/Examples/PubnubApi.ConsoleExample/packages.config

```diff
@@ -1,12 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="AsyncBridge.JetBrains" version="0.1.1.0" targetFramework="net40" />
-  <package id="Microsoft.Bcl" version="1.1.8" targetFramework="net461" />
-  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net461" />
-  <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net461" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net461" />
   <package id="Portable.BouncyCastle" version="1.8.1.2" targetFramework="net461" />
-  <package id="Pubnub" version="4.1.0" targetFramework="net461" />
+  <package id="Pubnub" version="4.1.1" targetFramework="net461" />
```

Comment:
I think this needs to be `4.2.0` as well right?

Reply:
updated.